content/articles/implementing-flask-api-using-linked-list/index.md

```diff
@@ -0,0 +1,428 @@
+### Implementing Flask API using Linked list Data Structure
+![hero_image](/engineering-education/implementing-flask-api-using-linked-list/hero.jpg)
```

Comment:
Hello, the hero image is not supposed to be in the article itself. It is only required in the article folder. Please delete this part.😊